README.md

```diff
@@ -41,6 +41,8 @@ this repository, you can try:
 
 * [Freenode #node.js channel](https://webchat.freenode.net/?channels=node.js&uio=d4)
 * [Gitter Node.js room](https://gitter.im/nodejs/node)
```

Comment:
Can you remove the Gitter line while we're in here?

Reply:
Done! 😄